### DISCUSSION TITLE
Wayland Issue on Fedora 39

### DISCUSSION THREAD

**Discussion by BerschBoss** [casualsnek/cassowary#146] [Q&A]
Hello,
I am a relatively new linux user, and I am really hopeful I can get this to run on my Fedora 39  installation.
I have followed the install instructions and got up to the point where I install cassowary and run it via python. Running 
`$ python3 -m cassowary -a`
netted me the following error:
`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.`
I see that it says to use QT_QPA_PLATFORM=wayland to potentially fix the issue, but I don't understand how I would go about 'using' that. Again, linux newbie, so I appreciate any help!

**Comment by tygoee** [comment]
To set `QT_QPA_PLATFORM`, just do:

`$ QT_QPA_PLATFORM=wayland python3 -m cassowary -a`

But I think you have to (re)install some library, look at [this qt issue](https://forum.qt.io/topic/93247/qt-qpa-plugin-could-not-load-the-qt-platform-plugin-xcb-in-even-though-it-was-found)
